### ISSUE TITLE
modeline has wrong foreground colors when emacs is started in daemon mode

### ISSUE THREAD

**Issue by vashirov** [syl20bnr/spacemacs#5398]
#### Description
I use ```emacsclient -c --alternate-editor=""``` to start my emacs. When started like this, modeline has wrong foreground colors. I can reproduce this on a clean system with the latest spacemacs from master.
#### Reproduction guide
- Start Emacs using ```emacsclient -c --alternate-editor=""```

*Observed behaviour:*
Foreground colors in the first section of modeline are wrong. 
![image](https://cloud.githubusercontent.com/assets/1745018/13581872/543ed52c-e4a9-11e5-933a-009c474f1003.png)

*Expected behaviour:*
Foreground colors in the first section of modeline should stay the same as when emacs is started normally.
![image](https://cloud.githubusercontent.com/assets/1745018/13581908/82ed1d3e-e4a9-11e5-9dab-d6e7cf185e56.png)

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.13
- Spacemacs branch: master (rev. 2c3e6b4)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(auto-completion emacs-lisp git org python shell syntax-checking version-control)
```

**Comment by sooheon** [comment]
Strange, I've had the exact same issue, but with a different repro method. For me, the method is:
1. add `themes-megapack`
2. `dotspacemacs-themes '(ample-flat)`

### Result
<img width="683" alt="screen shot 2016-03-09 at 12 44 24 am" src="https://cloud.githubusercontent.com/assets/302945/13606661/2cb98b0e-e590-11e5-9302-eaceb168fffb.png">

### Expected
<img width="683" alt="screen shot 2016-03-09 at 12 44 50 am" src="https://cloud.githubusercontent.com/assets/302945/13606667/3246471a-e590-11e5-8f39-7a19378f8601.png">

The way to get the expected result is to set dotspacemacs-themes to start with something supported like zenburn or spacemacs-light, and then load ample-flat theme.

**Comment by vashirov** [comment]
Seems to be working as expected in the latest develop

**Comment by TheBB** [comment]
Yes I fixed this in develop. Didn't know about this issue.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
